### ISSUE TITLE
Expose project via hook

### ISSUE THREAD

**Issue by andys8** [wbkd/react-flow#910]
In the past the react instance had to be used to access helper functions like `zoomIn`, `fitView` and `project`.

```
export type OnLoadParams<T = any> = {
  zoomIn: () => void;
  zoomOut: () => void;
  zoomTo: (zoomLevel: number) => void;
  fitView: FitViewFunc;
  project: ProjectFunc;
  getElements: () => Elements<T>;
  setTransform: (transform: FlowTransform) => void;
  toObject: ToObjectFunc<T>;
};
```
https://github.com/andys8/react-flow/blob/b408e6434dc0ddf92cbca4ed63b94ed3519b807e/src/types/index.ts#L252-L261

Now there is [`useZoomHelper`](https://reactflow.dev/docs/api/hooks/) which looks compelling.

I noticed `zoomIn`, `fitView` and similar functions are exposed, *but `project` isn't one of them*.

```
const initialZoomPanHelper: ZoomPanHelperFunctions = {
  zoomIn: () => {},
  zoomOut: () => {},
  zoomTo: (_: number) => {},
  transform: (_: FlowTransform) => {},
  fitView: (_: FitViewParams = { padding: DEFAULT_PADDING, includeHiddenNodes: false }) => {},
  setCenter: (_: number, __: number) => {},
  fitBounds: (_: Rect) => {},
  initialized: false,
};
```

https://github.com/andys8/react-flow/blob/b408e6434dc0ddf92cbca4ed63b94ed3519b807e/src/hooks/useZoomPanHelper.ts#L11-L20

## Feature Request

Could `project` be added to this or a new hook to be exposed the same way?
The advantage would be, that it's not necessary to access the react flow instance anymore.

**Comment by moklick** [comment]
That's a good idea! I will add it in v9.0.0. If you have more ideas/ needs for certain hooks let us know. 

Btw if you are running a project with react-flow, could you try to install `react-flow-renderer@next` to test if everything works as expected? Would be helpful for us :)

**Comment by moklick** [comment]
The hook is now available when you install `react-flow-renderer@next`

**Comment by moklick** [comment]
released in v9.0.0
